tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Catalogue Solutions Get All.feature

```diff
@@ -0,0 +1,46 @@
+﻿Feature: Catalogue Solutions Get All
```

Comment:
Maybe change it to something a bit more meaningfull

Display all of the catalogue solutions for an order